RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -335,7 +335,17 @@ class PumpOpsSynchronous {
 
         pages: for pageNum in 0..<16 {
             NSLog("Fetching page %d", pageNum)
-            let pageData = try getHistoryPage(pageNum)
+            let pageData: NSData
+
+            do {
+                pageData = try getHistoryPage(pageNum)
+            } catch let error as PumpCommsError {
+                if pageNum > 0 {
```

Comment:
Yeah, that's what I was wondering; I left my empty history 522 at home, so I can't test.